src/main/webapp/i18n/en/feature.json

```diff
@@ -19,7 +19,19 @@
       "datasourceId": "Datasource Id",
       "dateFilter": "Date Filter",
       "definition": "Definition",
-      "favouriteFilter": "Favourite Filter"
+      "favouriteFilter": "Favourite Filter",
+      "panel": {
+        "create": "+"
```

Comment:
for icons use https://react-spectrum.adobe.com/react-spectrum/workflow-icons.html
available icon https://spectrum.adobe.com/page/icons/